docs/manual/hacking.rst

```diff
@@ -18,6 +18,13 @@ On Ubuntu, if testing on Python 3, this can be done with:
 
     sudo apt-get install python3-pytest xserver-xephyr x11-apps
 
+On ArchLinux, the X11 requirements are installed with:
+
+.. code-block:: bash
+
+    sudo pacman -S xorg-xrandr xorg-xcalc xorg-xeyes xorg-xclock
+
+
```

Comment:
Not sure if you're willing to maintain these distro-dependent installation methods. There already was Ubuntu though, so I thought why not adding Arch.